lib/bundler/friendly_errors.rb

```diff
@@ -89,8 +89,10 @@ def request_issue_report_for(e)
     end
 
     def issues_url(exception)
+      message = exception.message.lines.first.tr(":", " ").chomp
+      message = message.split("-").first if exception.is_a?(Errno::EACCES)
```

Comment:
this could happen on errors other than an `EACCESS` too, I believe, such as `ENOTSUP`